.env.development.native

```diff
@@ -29,4 +29,5 @@ REACT_APP_AUTH0_CLIENT_ID=tkEmIIdd8eAkLGSnNWcRdA0ARI92NxHr
 REACT_APP_AUTH0_SMS_CLIENT_ID=421OOR4Y4BBUwyC76H5ajLP4GpY7ms7A
 REACT_APP_FACEBOOK_APP_ID=2494863677435036
 REACT_APP_GOOGLE_CLIENT_ID=1015336103925-reqktqs0ns9vfaeh7nbt8mi634u9157k.apps.googleusercontent.com
-REACT_APP_ZOOM_LICENSE_KEY=d41aVNPcjbLOYm5IKHzenHk0rnGBWkiz
\ No newline at end of file
+REACT_APP_ZOOM_LICENSE_KEY=d41aVNPcjbLOYm5IKHzenHk0rnGBWkiz
+REACT_APP_NFT_STORAGE_KEY="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJkaWQ6ZXRocjoweEMxQzUwRjJkNUYwNzgxNDJEOEEyRGQ3ZkQ4NUY5ODVFMzZEOUI5NzAiLCJpc3MiOiJuZnQtc3RvcmFnZSIsImlhdCI6MTYyMjQ2NTYyNzMyMCwibmFtZSI6Ikdvb2REb2xsYXJUZXN0In0.9Q02zYBwlJymF6ognSrvqdzch1Vo-Xw27QYA-ABlJsY"
```

Comment:
this shouldnt be commited.....

Reply:
there are dev secret linked to my personal nft.storage account so it's ok. as you could see - the FV key from theirs test API is commited here too